XamlControlsGallery/ControlPagesSampleCode/ListView/ListViewSample4_xaml.txt

```diff
@@ -0,0 +1,27 @@
+<!-- XAML Code -->
+
+<!-- The ListView's ItemsSource is an ObservableCollection<Contact> that is defined and bound
+in the code-behind, called contactsFiltered. This ListView uses the same DataTemplate as the
+samples above it, called ContactListViewTemplate - you can see its definition above. -->
+
+<ListView 
+    x:Name="FilteredListView"
+    ItemTemplate="{StaticResource ContactListViewTemplate}"
+    SelectionMode="Single"
+    ShowsScrollingPlaceholders="True"
+    Height="400"
+    Width="400"
+    BorderThickness="1"
+    BorderBrush="{ThemeResource SystemControlForegroundBaseMediumLowBrush}">
+</ListView>
```

Comment:
back to my nit about hanging closing brackets :)

Reply:
One day I'll learn...